empire/server/authorization/github/github.go

```diff
@@ -22,6 +22,9 @@ type Authorizer struct {
 	// organization.
 	Organization string
 
+	// The oauth application URL.
+	ApiUrl string
```

Comment:
Minor: can we capitalize `URL` here? We try to follow [this convention](https://github.com/golang/go/wiki/CodeReviewComments#initialisms).